distributed/scheduler.py

```diff
@@ -837,6 +837,7 @@ def __init__(
 
         # Worker state
         self.workers = SortedDict()
+        self.workers_values = self.workers.values()
```

Comment:
Why this change?  Is this expensive?  More expensive than converting it to a set?